tests/c_api_tests/test_c_api_vnf_converter.cpp

```diff
@@ -0,0 +1,53 @@
+// SPDX-FileCopyrightText: Contributors to the Power Grid Model project <powergridmodel@lfenergy.org>
+//
+// SPDX-License-Identifier: MPL-2.0
+
+#include <power_grid_model_io_native_c/basics.h>
+#include <power_grid_model_io_native_c/handle.h>
+#include <power_grid_model_io_native_c/vnf_pgm_converter.h>
+
+#include <power_grid_model/common/exception.hpp>
+
+#include <cstring>
+#include <doctest/doctest.h>
+
+namespace power_grid_model_io_native {
+
+TEST_CASE("Test PGM_IO_create_vnf_converter") {
+    PGM_IO_Idx experimental_feature_flag = 0;
+
+    SUBCASE("Test PGM_IO_create_vnf_converter without experimental feature flag") {
+        PGM_IO_Handle* handle0 = PGM_IO_create_handle();
+        auto converter = PGM_IO_create_vnf_converter(handle0, nullptr, experimental_feature_flag);
+        CHECK(PGM_IO_error_code(handle0) == PGM_IO_regular_error);
+        PGM_IO_destroy_vnf_converter(converter);
+        PGM_IO_destroy_handle(handle0);
+    }
+
+    SUBCASE("Test PGM_IO_create_vnf_converter with experimental feature flag") {
+        PGM_IO_Handle* handle1 = PGM_IO_create_handle();
```

Comment:
you can re-use the same variable name
```suggestion
        PGM_IO_Handle* handle = PGM_IO_create_handle();
```